resources/definitions/leapfrog_bolt_pro.def.json

```diff
@@ -0,0 +1,131 @@
+        "has_machine_quality": true,
+        "has_variants": true,
+        "preferred_variant_name": "Brass 0.4",
+        "preferred_material": "leapfrog_epla_natural",
+        "variants_name": "Hot end",
+        "exclude_materials": [
+            "generic_pla",
+            "generic_pla_175",
+            "generic_abs",
+            "generic_abs_175",
+            "generic_bam",
+            "generic_cpe",
+            "generic_cpe_175",
+            "generic_cpe_plus",
+            "generic_hips",
+            "generic_hips_175",
+            "generic_nylon",
+            "generic_nylon_175",
+            "generic_pc",
+            "generic_pc_175",
+            "generic_petg",
+            "generic_petg_175",
+            "generic_pp",
+            "generic_pva",
+            "generic_pva_175",
+            "generic_tough_pla",
+            "generic_tpu",
+            "generic_tpu_175",
+            "chromatik_pla",
+            "dsm_arnitel2045_175",
+            "dsm_novamid1070_175",
+            "emotiontech_abs",
+            "emotiontech_petg",
+            "emotiontech_pla",
+            "emotiontech_pva-m",
+            "emotiontech_pva-oks",
+            "emotiontech_pva-s",
+            "emotiontech_tpu98a",
+            "fabtotum_abs",
+            "fabtotum_nylon",
+            "fabtotum_pla",
+            "fabtotum_tpu",
+            "fiberlogy_hd_pla",
+            "filo3d_pla",
+            "filo3d_pla_green",
+            "filo3d_pla_red",
+            "imade3d_petg_175",
+            "imade3d_pla_175",
+            "innofill_innoflex60_175",
+            "octofiber_pla",
+            "polyflex_pla",
+            "polymax_pla",
+            "polyplus_pla",
+            "polywood_pla",
+            "tizyx_abs",
+            "tizyx_pla",
+            "tizyx_flex",
+            "tizyx_petg",
+            "tizyx_pva",
+            "tizyx_pla_bois",
+            "verbatim_bvoh_175",
+            "Vertex_Delta_ABS",
+            "Vertex_Delta_PET",
+            "Vertex_Delta_PLA_Glitter",
+            "Vertex_Delta_PLA_Mat",
+            "Vertex_Delta_PLA_Satin",
+            "Vertex_Delta_PLA_Wood",
+            "Vertex_Delta_PLA",
+            "Vertex_Delta_TPU",
+            "zyyx_pro_flex",
+            "zyyx_pro_pla"
+        ],
+        
+        "machine_extruder_trains":
+        {
+            "0": "leapfrog_bolt_pro_extruder_right",
+            "1": "leapfrog_bolt_pro_extruder_left"
+        }
+    },
+    "overrides": {
+        "machine_name": {"default_value": "Leapfrog Bolt Pro" },
+        "machine_extruder_count": {"default_value": 2},
+		"machine_center_is_zero": {"default_value": false},
+        "machine_width": {"default_value": 302},
+        "machine_height": {"default_value": 205},
+        "machine_depth": {"default_value": 322},
+        "machine_heated_bed": {"default_value": true},
+        "machine_head_with_fans_polygon": {"default_value": [[-1, 1 ], [1, 1], [1, -1 ], [-1, -1]]},

```

Comment:
Note that this might cause damage to printer or prints if a user prints with one at a time.

Reply:
Thank you for your feedback. The extruder heads are ''homed'' outside of the build volume. When only one of the extruders used, this one will be printing within the build volume while the other tool will be inactive on its homing location outside of the build volume.

Reply:
Printing one at the time has little to do with homing, but with knocking a print over because they are not printed one layer at a time. So if you have 2 objects on the buildplate (called A & B) and all of them are printed with the same extruder, it will first completely print A and then print B.

This means that some avoidance needs to be done in order to prevent colisisons, which is calculated by this property. It's fine by me to leave it, but i just wanted to make sure that this is intentional.

Reply:
Dear Nallath, Thank you for clarifying this . We have adjusted the values (also gantry height).